### ISSUE TITLE
Issues when running fine tuning of pre-trained CGCNNs

### ISSUE THREAD

**Issue by ngkayjay** [zcao0420/MOFormer#5]
Running `python finetune_cgcnn.py` calls dataset_finetune_cgcnn.py:
But in MOFormer/dataset/dataset_finetune_cgcnn.py line 14, an import was made from `fine_tuning` but no such module or package seems to exist.

**Comment by ngkayjay** [comment]
Also, it appears that fine-tuning benchmark datasets for the CGCNN are not provided (root_dir `./QMOF_cg` and `./hMOF_cif` do not exist). I suppose this one might be intended behavior.

**Comment by zcao0420** [comment]
Thank you catching that import error. This is again another example of legacy code that was missed before we made this repo public. It's now fixed. 

As for the datasets, we did not upload them since they are already publicly available. I would recommend you to follow those links listed in the Acknowledgement section for the original papers to hMOF and QMOF. BTW, the `CGData` class in the `dataset_finetune_cgcnn.py` can be used to directly load converted crystal graphs. Since converting CIF files to crystal graph is usually the speed bottleneck when training CGCNN on large dataset, converting and storing the crystal graphs can be useful when you are running finetuning multiple times.
